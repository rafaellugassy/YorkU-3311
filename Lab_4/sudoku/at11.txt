set_number(1, 1, 1)
set_number(2, 2, 2)
set_number(3, 3, 3)
set_number(4, 4, 4)
set_number(3, 1, 4)
set_number(4, 2, 3)
set_number(1, 3, 2)
set_number(2, 4, 1)

start_game

put_number(4, 1, 2)
put_number(3, 4, 2)
put_number(3, 2, 1)
put_number(4, 3, 1)
put_number(1, 2, 4)
put_number(2, 3, 4)
put_number(2, 1, 3)
put_number(1, 4, 3)

restart_game
