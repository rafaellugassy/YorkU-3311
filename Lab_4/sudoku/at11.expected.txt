  Setting up a new grid
  0   0   0   0
  0   0   0   0
  0   0   0   0
  0   0   0   0
->set_number(1,1,1)
  Setting up a new grid
  1   0   0   0
  0   0   0   0
  0   0   0   0
  0   0   0   0
->set_number(2,2,2)
  Setting up a new grid
  1   0   0   0
  0   2   0   0
  0   0   0   0
  0   0   0   0
->set_number(3,3,3)
  Setting up a new grid
  1   0   0   0
  0   2   0   0
  0   0   3   0
  0   0   0   0
->set_number(4,4,4)
  Setting up a new grid
  1   0   0   0
  0   2   0   0
  0   0   3   0
  0   0   0   4
->set_number(3,1,4)
  Setting up a new grid
  1   0   0   3
  0   2   0   0
  0   0   3   0
  0   0   0   4
->set_number(4,2,3)
  Setting up a new grid
  1   0   0   3
  0   2   4   0
  0   0   3   0
  0   0   0   4
->set_number(1,3,2)
  Setting up a new grid
  1   0   0   3
  0   2   4   0
  0   1   3   0
  0   0   0   4
->set_number(2,4,1)
  Setting up a new grid
  1   0   0   3
  0   2   4   0
  0   1   3   0
  2   0   0   4
->start_game
  Current game is over: No
  1   0   0   3
  0   2   4   0
  0   1   3   0
  2   0   0   4
->put_number(4,1,2)
  Current game is over: No
  1   4   0   3
  0   2   4   0
  0   1   3   0
  2   0   0   4
->put_number(3,4,2)
  Current game is over: No
  1   4   0   3
  0   2   4   0
  0   1   3   0
  2   3   0   4
->put_number(3,2,1)
  Current game is over: No
  1   4   0   3
  3   2   4   0
  0   1   3   0
  2   3   0   4
->put_number(4,3,1)
  Current game is over: No
  1   4   0   3
  3   2   4   0
  4   1   3   0
  2   3   0   4
->put_number(1,2,4)
  Current game is over: No
  1   4   0   3
  3   2   4   1
  4   1   3   0
  2   3   0   4
->put_number(2,3,4)
  Current game is over: No
  1   4   0   3
  3   2   4   1
  4   1   3   2
  2   3   0   4
->put_number(2,1,3)
  Current game is over: No
  1   4   2   3
  3   2   4   1
  4   1   3   2
  2   3   0   4
->put_number(1,4,3)
  Current game is over: Yes
  1   4   2   3
  3   2   4   1
  4   1   3   2
  2   3   1   4
->restart_game
  Setting up a new grid
  0   0   0   0
  0   0   0   0
  0   0   0   0
  0   0   0   0
